app/models/form_profiles/va_526ez.rb

```diff
@@ -92,12 +92,17 @@ def metadata
   def initialize_rated_disabilities_information
     return {} unless user.authorize :evss, :access?
 
-    service = ApiProviderFactory.rated_disabilities_service_provider(
-      {
-        auth_headers: EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h),
-        icn: user.icn.to_s
-      }
+    service = ApiProviderFactory.call(
+      type: ApiProviderFactory::FACTORIES[:rated_disabilities],
+      provider: nil,
+      options: {
+        icn: user.icn.to_s,
+        auth_headers: EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+      },
+      current_user: nil,
+      feature_toggle: ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_FOREGROUND
     )
+
     response = service.get_rated_disabilities
```

Comment:
Here `service` is the result of the service, not an instance of the service, because of the `call` patter we used.  Can we update this variable name to something like `api_provider`?